CHANGELOG.md

```diff
@@ -298,13 +298,17 @@ You can now add attributes like classes, rowspan and colspan to table row header
 
 [Pull request #1367: Allow for classes, rowspan, colspan and attributes on row headers](https://github.com/alphagov/govuk-frontend/pull/1367). Thanks to [edwardhorsford](https://github.com/edwardhorsford).
 
-### Use page wrappers without a modifier
+### Use page wrapper auto spacing
 
-If your page wrapper does not have components above the main section, you no longer need to use the [`govuk-main-wrapper--l` modifier class](https://design-system.service.gov.uk/styles/layout/#exploded-view-of-page-wrappers-without-a-back-link-breadcrumbs-or-phase-banner) to increase the vertical space above the content.
+You can now use the the `.govuk-main-wrapper--auto-spacing` modifier on the main wrapper.
+
+This will apply the correct spacing depending on whether there are any elements (such the back link, breadcrumbs or phase banner components) before the `.govuk-main-wrapper` in the `.govuk-width-container`.
```

Comment:
More tweaks to match the language. Also removed brackets, as we should try to avoid those wherever poss. Let me know if still reads ok.

```suggestion
This will add the correct amount of padding depending on whether there are elements above the `<main>` element inside the `govuk-width-container` wrapper, like the back link, breadcrumbs or phase banner components.
```

Reply:
Good to know about brackets thanks Mark